# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLMethodDispatcher.h
# Commit: 5005fe56b7b0
# Full Hash: 5005fe56b7b0153d2069d25c6f1f4e311e2c1eb1
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-02-21 05:56:31
# Regressor Bug: 1616715
# File Overlap Count: 1
# Description:
#   Bug 1616715 - SurfaceFromElement gets alpha Premult unless opt-in to NonPremult. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D63421
# ==============================================================================

diff -r 985dccd11e97 -r 5005fe56b7b0 dom/canvas/WebGLMethodDispatcher.h
--- a/dom/canvas/WebGLMethodDispatcher.h	Thu Feb 20 21:12:05 2020 +0000
+++ b/dom/canvas/WebGLMethodDispatcher.h	Thu Feb 20 20:52:10 2020 +0000
@@ -52,7 +52,6 @@
 DEFINE_ASYNC(HostWebGLContext::DeleteVertexArray)
 
 DEFINE_ASYNC(HostWebGLContext::ClearVRFrame)
-DEFINE_SYNC(HostWebGLContext::GetSurfaceSnapshot)
 DEFINE_SYNC(HostWebGLContext::GetVRFrame)
 
 DEFINE_ASYNC(HostWebGLContext::Disable)